common/compositor/va/varenderer.cpp

```diff
@@ -156,6 +156,9 @@ bool VARenderer::GetVAProcDeinterlaceFlagFromVideo(HWCDeinterlaceFlag flag) {
   } else {
     // TODO:Need video buffer meta data to judge if the frame really need
     // Deinterlace.
+    if (OverlayBuffer::GetInterlaceMode() != kDeinterlaceNone) {
+      return true;
```

Comment:
The logic is wrong here.
OverlayBuffer::GetInterlaceMode() shoud not be static function
We should able to track is interlaced fied regarding to each overlay buffer instance

Reply:
It also can't be compiled